.eslintrc.json

```diff
@@ -36,6 +30,13 @@
     "react/button-has-type": "warn",
     "react/no-array-index-key": "warn",
     "react/no-render-return-value": 0,
+    "react/boolean-prop-naming": [
+      "error",
+      {
+        "rule": "^(is)[A-Z]([A-Za-z0-9]?)+",
+        "message": "It is better if your prop ({{ propName }}) starts with `is` or `has`"
```

Comment:
"It is better if" does not imply error :eyes: 
Should be stronger, like "Your prop .... should start blah blah"